events/docker/docker-compose.yml

```diff
@@ -37,25 +40,22 @@ services:
         # cpu_count: 1
         # mem_limit: 900m
         labels:
-            - "traefik.backend=events"
-            - "traefik.port=8084"
-            - "traefik.auth.frontend.rule=PathPrefix:/api/events/metrics;PathPrefixStrip:/api/events;"
-            - "traefik.old.frontend.rule=PathPrefixStrip:/services/oms-events/api;"
-            - "traefik.new.frontend.rule=PathPrefixStrip:/api/events;"
-            - "traefik.auth.frontend.auth.basic.users=admin:${METRICS_CREDENTIALS}"
-            - "traefik.old.frontend.priority=110"
-            - "traefik.new.frontend.priority=110"
-            - "traefik.auth.frontend.priority=120"
             - "traefik.enable=true"
+            - "traefik.http.routers.events.entrypoints=websecure"
+            - "traefik.http.routers.events.tls=true"
+            - "traefik.http.routers.events.rule=Host(`${SUBDOMAIN_FRONTEND}${BASE_URL}`) && PathPrefix(`/api/events`)"
+
+            - "traefik.http.routers.events.middlewares=events-stripprefix"
+            - "traefik.http.middlewares.events-stripprefix.stripprefix.prefixes=/api/events"
+
+            - "traefik.http.routers.metrics.rule=Host(`${SUBDOMAIN_FRONTEND}${BASE_URL}`) && PathPrefix(`/api/events/metrics`)"
+            - "traefik.http.routers.metrics.middlewares=events-stripprefix, metricsauth"
 
     events-static:
         restart: on-failure
-        image: aegee/nginx-static:latest
+        image: aegee/nginx-static:prod
```

Comment:
Does this mean we also updated nginx versions? Not sure if we really maintained this image. Which is the Dockerfile for it?

Reply:
i only made a Dockerfile FROM aegee/nginx-static:latest and with the 3 COPY instructions, and pushed